plugins/status/plugin.go

```diff
@@ -454,3 +467,29 @@ func (p *Plugin) logUpdate(update *UpdateRequestV1) error {
 	}).Info("Status Log")
 	return nil
 }
+
+func updatePrometheusMetrics(ctx context.Context, u *UpdateRequestV1) {
```

Comment:
[nit]
```suggestion
func updatePrometheusMetrics(_ context.Context, u *UpdateRequestV1) {
```

🔍 It looks like we could just as well drop the parameter -- we're the only callers here.